config/config.yaml

```diff
@@ -50,6 +50,11 @@ assets:
         link: '%css_header%'
         autoload: true
         allowNoLoad: true
+    compact-theme:
+        alreadyBuilt: true
+        link: '%compact_header%'
+        autoload: true
+        allowNoLoad: true
```

Comment:
To avoid confusion, recommend remove the 1 line above and place a comment like:
# This asset follows the allowNoLoad setting of the main-theme asset